README.md

```diff
@@ -2,9 +2,18 @@
 
 [![configlet](https://github.com/exercism/8th/workflows/configlet/badge.svg)](https://github.com/exercism/8th/actions?query=workflow%3Aconfiglet) [![tests](https://github.com/exercism/8th/workflows/test/badge.svg)](https://github.com/exercism/8th/actions?query=workflow%3Atest)
 
-Exercism exercises in 8th.
+Exercism exercises in [8th](https://8th-dev.com/manual.html).
 
-## Testing
+## CONTRIBUTING
+
+### Caution: CI will fail for PRs coming from forked repos
+
+This track [uses its own test runner for CI](https://github.com/exercism/8th/blob/1176bfbd9457f51556c77b8cc26408d718d1b4d6/.github/workflows/test.yml#L18).
+This requires a secret token to grant access.
+I'm not sure how this mechanism works, but it means that, for PRs originating from _a fork of this repo_, access cannot be granted and CI will fail: CI will only pass for PRs merging an exercism/8th branch.
```

Comment:
Maybe remove the I'm not sure part?

Reply:
Is the rest of it ("it means that ...") more or less correct?

Reply:
It is. Forks don't have access to secrets for security reasons, hence the workflow thinking there is no token provided, as that is using a secret.